core/backend/src/IModelDb.ts

```diff
@@ -2768,8 +2768,22 @@ export class BriefcaseDb extends IModelDb {
         briefcaseDb._codeService = await CodeService.createForIModel(briefcaseDb);
         this.onCodeServiceCreated.raiseEvent(briefcaseDb);
       } catch (e: any) {
-        if ((e as CodeService.Error).errorId !== "NoCodeIndex") // no code index means iModel isn't enforcing codes.
-          throw e;
+        if ((e as CodeService.Error).errorId !== "NoCodeIndex") { // no code index means iModel isn't enforcing codes.
```

Comment:
I don't know what the exception string should be. It should probably explain that the codeservice is not available and append the string from `e`. But I'm wondering where this comes up. Didn't we make the CodeService implementation not throw for authorization errors? @johnnyd710?

Reply:
@swwilson-bsi was the one who originally reported this so maybe he would have some ideas about how this occurred.

Reply:
I made it throw the "user is not authorized to access CodeService" error (thrown when a user does not have all necessary RBAC permissions) when the code service is actually used, not on initialization. See https://github.com/iTwin/code-service/pull/9

Reply:
@kabentley Hi, are we okay to resolve this conversation?

Reply:
Yes, you can resolve this